source/_integrations/zha.markdown

```diff
@@ -233,6 +233,23 @@ To see OTA updates for a device, it's required that it both supports OTA updates
 Before updating a device, you should search for any disadvantages or if you even need to install an available update. Some firmware updates can break features you might use (e.g. group binding for IKEA devices). Some updates might also require changes to ZHA. In rare cases, you can even brick devices by installing a firmware update.
 {% endwarning %}
 
+### Global Options
+
+There are a few global options available once ZHA has been configured. Press **Configure** to access these settings.
+
+The options are as follows:
+
+{% configuration_basic %}
+Enable enhanced light color/temperature transition from an off-state:
+  description: "For older non Zigbee 3.0 lights, this still allows a proper transition from an off-state to a new color (without seeing the old color). _(default: off)_"
```

Comment:
@TheJulianJES for this option, what is the downside to turning it on? I think we turn it off for a reason.

Reply:
If the light is off and the color/temp is changed in the "on" call, an additional message will be sent.
On some networks, this can cause issues, especially with ZHA group lights. Still, the option does work fine for others.

Generally, it's recommended to set the `options` attribute on the `Color` cluster to enable "send color while off", but that only works on Zigbee 3.0 lights and the UX isn't great (yet).